Build/BuildScripts/CodeJam.AppVeyor.NUnit.Tests.ps1

```diff
@@ -27,7 +27,7 @@ foreach ($target in $targetsDotNet) {
 }
 
 #run .net core tests
-$targetsDotNetCore = "net5.0;netcoreapp3.1;netcoreapp3.0;netcoreapp2.1;netcoreapp1.1" -split ";"
+$targetsDotNetCore = "net5.0;netcoreapp3.1" -split ";"
```

Comment:
If we support .NET Standard 2.0 it means we support .NET Core 2.0 and above, so need to test it in unit tests.
https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md

Reply:
And if test failed under Core 2.0 specifically, what should we do in this case?

Reply:
If it is some issue in .NET Core 2.0 which we can workaround , then we add a workaround.

Reply:
If we do not have a separate build for this version, where should we add a workaround?

Reply:
Currently we test (almost) all possible consumers: 
.NET Core 3.1, .NET Core 3.0, .NET Core 2.1, can also add .NET Core 2.0.
This covers all .NET Standard 2.0 supporting Core frameworks.

Do you say 3.1 is enough to make sure everything is working for older ?
Perhaps we should then test on 3.1 and 2.0 and hope nothing is broken in the versions between :)